packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -226,13 +232,20 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                             ) : (
                                 <div
                                     className='p2p-my-ads__table-add'
-                                    onClick={() => {
-                                        setAdvert(advert);
-                                        my_ads_store.showQuickAddModal(advert);
-                                    }}
+                                    onClick={
+                                        !general_store.is_barred &&
+                                        (() => {
```

Comment:
Since the function body is same, you can declare it above and call it here. Also refer to the same comment above

Reply:
Thx for the suggestion! Will do that